project/models/project_base.py

```diff
@@ -1181,6 +1181,14 @@ def get_artif_per_zone_urba_type(
             str | float,
         ],
     ]:
+        zone_labels = {
+            "U": "zone urbaine",
+            "AUc": "zone à urbaniser",
+            "AUs": "zone à urbaniser bloquée",
+            "A": "zone agricole",
+            "N": "zone naturelle",
+        }
```

Comment:
```suggestion
        zone_labels = {
            "U": "zone urbanisée",
            "AUc": "zone à urbaniser alternatif",
            "AUs": "zone à urbaniser bloquée",
            "A": "zone agricole",
            "Ah": "zone agricole constructible",
            "N": "zone naturelle et forestière",
            "Nh": "zone naturelle constructible",
        }
```
j'ai repris les libellés qu'utilise le GPU : https://www.geoportail-urbanisme.gouv.fr/api/#/Categories/get_du_categories